bundle/schema/spec.go

```diff
@@ -0,0 +1,210 @@
+package schema
```

Comment:
Can we move this to a separate library (ie `libs/openapi`)?

Reply:
Actually, on further thought, it's going to be a really small struct. We can keep it as in and likely don't need a separate library yet.

Reply:
👍